compiler/damlc/daml-stdlib-src/LibraryModules.daml

```diff
@@ -52,4 +54,9 @@ import DA.Time
 import DA.Traversable
 import DA.Tuple
 import DA.Validation
+
+#ifdef DAML_EXPERIMENTAL
+import DA.Experimental.Example
+#endif
```

Comment:
In general I prefer to do this the other way: don't use CPP in LibraryModules, and use it only in DA.Experimantal.Example instead, where you export an empty module if the feature flag is not available. This is how it's done everywhere else in the standard library. (See DA.Map for example.)

Reply:
I dislike the fact that we will end up with empty `DA.Experimental` module in the non dev standard library. 
This is specially hugely for something that will not be in future version.

Reply:
Ok, makes sense.